docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -23,15 +23,21 @@ description: "Sample for supplying a WebLogic Deploy Tool (WDT) model that the o
 
 ### Introduction
 
-This sample demonstrates:
+This sample demonstrates deploying a Model in Image domain home source type. Unlike the Domain in PV and Domain in Image domain source types, Model in Image eliminates the need to pre-create your WebLogic domain home prior to deploying your domain resource. Instead, a Model in Image deployment uses a WebLogic Deploy Tool (WDT) model to specify your WebLogic configuration. 
```

Comment:
Unlike the Domain in PV and Domain in Image domain source types, -> Unlike Domain in PV and Domain in Image,

Reply:
@rosemarymarano Done.  Please resolve the comment and approve the pull you're OK with this.

Reply:
ok